components/board/analog_smoother.go

```diff
@@ -87,6 +91,10 @@ func (as *AnalogSmoother) Start(ctx context.Context) {
 	//    numSamples        4
 
 	numSamples := (as.SamplesPerSecond * as.AverageOverMillis) / 1000
+	if numSamples <= 0 {
```

Comment:
if numSamples is zero, do we really need to instantiate a rolling average at all? Just report that values and skip that part of the code.

Reply:
I would prefer this behavior. This allows the user to access the raw values as often as they please.

Reply:
Tell me more. Wouldn't skipping that part of the code require extra complexity? Like, "if numSamples is at most 1, do one thing, and otherwise do something completely different"? If I were to implement what I think your suggestion is, this PR will balloon from an 8-line change to a 30-line change. Perhaps I misunderstood?

Reply:
and log that you're getting it as is.

Reply:
the num samples isn't the cause of a division by zero error anywhere I can see, so instead of calling the RollingAverage function on line 98, you're enforce a rolling average of 1 with this behavioural change.

Reply:
The div by zero is a couple lines below: `	nanosBetween := 1e9 / as.SamplesPerSecond`

Regardless I agree with your solution; if NumSamples <= 0, then the raw value should be returned.